### PR TITLE
Add Genexus detection

### PR DESCRIPTION
**Pull request by GonzaloRizzo**
This adds support for Genexus detection.

It's a fast web and mobile application generation solution mainly focused on enterprise solutions.

I don't know how popular it is in other countries, but here in Uruguay it's used on almost every mid sized business and on public entities related software.

It seems used by a lot of other international business according to their page.

https://www.genexus.com/en/global/company/success-stories
https://www.genexus.com/en/global/community/partners/solution-partners

I can't give you a repo since because of its enterprise nature it's strongly closed software, you can think about it as an Uruguayan Ballmer era Microsoft

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -12285,6 +12285,19 @@
...
+        18, 19
+      ],
+      "icon": "genexus.png",
+      "website": "https://www.genexus.com",
+      "js": {
+        "gx": "",
+        "gx.evt": ""
+      },
+      "html": "<[^>]+class=\"gxp-page\"",
+      "script": "gxp\\.js"
```
Comment (jvoisin) [inline src/apps.json]: I think that using `/gxp\\.js` might reduce the number of false-positives :)

File: src/apps.json [hunk 2]
```diff
@@ -12285,6 +12285,19 @@
...
       "website": "https://woosa.nl"
+    },
+    "Genexus": {
+      "cats": [
+        18, 19
+      ],
+      "icon": "genexus.png",
+      "website": "https://www.genexus.com",
+      "js": {
+        "gx": "",
```
Comment (gadcam) [inline src/apps.json]: <code>gx</code> alone is too false positive prone : a lot of JS packers generate short variable names. I think the best is just to remove it as you already added <code>gx.evt</code>

Comment (GonzaloRizzo) [reply src/apps.json]: Yes, you are right

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**